vmx/tests/box.rs

```diff
@@ -19,7 +19,7 @@ pub fn box_set_get() -> Result<(), Error> {
     let mut session = vm.session();
 
     session.transact::<i16, ()>(id, "set", 0x11)?;
-    session.commit(&id)?;
+    session.commit()?;
```

Comment:
Commit here should not be necessary, since the session has already mutated the box content.

Reply:
done